relationships/synthesis/EXT-SERVICE-to-MEMCACHED.yml

```diff
@@ -66,8 +66,8 @@ relationships:
           fields:
             - field: endpoint
               attribute: net.peer.name
-            - field: port
-              attribute: 11211 # Default port for memcached
+#            - field: port
+#              attribute: 11211 # Default port for memcached
```

Comment:
imho this and removing the rule is the same.

Now we will send a candidate that doesn't match anything never because it requires endpoint and port to match: https://github.com/newrelic/entity-definitions/blob/main/relationships/candidates/MEMCACHED.yml